python/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ if(ARB_USE_BUNDLED_PYBIND11)
   # instead of using find_package.
   add_subdirectory(${pb11_src_dir} pybind11)
 else()
-  find_package(pybind11 2.8.1 REQUIRED)
+  find_package(pybind11 2.10.0 REQUIRED)
```

Comment:
Sneaking version bump?
We agreed to separate these out into their own PRs.

Reply:
A PR was merged in my absence with an outdated version number here. Pybind11 was bumped over a month ago to 2.10.0

Reply:
Ah!